apps/app-article/src/App.js

```diff
@@ -68,10 +68,23 @@ class App extends Component {
             this.setState({fontSize: parseFloat(localStorage.getItem("fontSize"))});
         }
 
+        if(getCookie('LoggedOutFlag')){
+            let d = new Date();
+            d.setTime(d.getTime() - (1000*60*60*24)); //Set the time to the past. 1000 milliseonds = 1 second
+            let expires = "expires=" + d.toUTCString();
+            window.document.cookie = "LoggedOutFlag=" + "; "+expires;
+            logout(this.onLogout);
+            localStorage.removeItem('token');
```

Comment:
Calling `User.logout` will clean up the local storage from `uuid` and `token`, so you don't need to do this.